tests/unit/EntityId/PrefixMappingEntityIdParserTest.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace Wikibase\DataModel\Services\Tests\EntityId;
+
+use Wikibase\DataModel\Entity\BasicEntityIdParser;
+use Wikibase\DataModel\Entity\EntityIdParsingException;
+use Wikibase\DataModel\Entity\EntityId;
+use Wikibase\DataModel\Entity\ItemId;
+use Wikibase\DataModel\Entity\PropertyId;
+use Wikibase\DataModel\Services\EntityId\PrefixMappingEntityIdParser;
+use Wikimedia\Assert\ParameterAssertionException;
+use Wikimedia\Assert\ParameterTypeException;
+
+/**
+ * @covers Wikibase\DataModel\Services\EntityId\PrefixMappingEntityIdParser
+ *
+ * @license GPL-2.0+
+ */
+class PrefMappixingEntityIdParserTest extends \PHPUnit_Framework_TestCase {
```

Comment:
I like "Mappixing", but the name of the test class should be consistent with the class under test :)